tests/common.sh

```diff
@@ -6,8 +6,12 @@ TMP="$(mktemp -d)"
 trap "rm -rf $TMP" EXIT
 
 export SNAPPY_LAUNCHER_SECCOMP_PROFILE_DIR="$TMP"
+export SNAPPY_LAUNCHER_ENVIRONMENT_DIR="$TMP/environment"
```

Comment:
I think we should stop doing this and just define SNAPPY_LAUNCHER_something_something_ROOT and have everything else be relative from there. We're getting mount profiles next (another directory, another variable).